.ci/Jenkinsfile

```diff
@@ -136,7 +136,9 @@ pipeline {
             dir(BASE_DIR){
               unstash 'docker'
               sh script: '.ci/scripts/tag-metricbeats.sh', label: 'Create docker tag'
-              sh script: '.ci/scripts/functional-test.sh', label: 'Run functional tests'
+              sh script: """#!/bin/bash
+              .ci/scripts/functional-test.sh "${GO_VERSION}" "${FEATURE}"
```

Comment:
This ${FEATURE} comes empty by default, right?

Reply:
indeed